docs/reference/api/docker_remote_api.md

```diff
@@ -24,6 +24,17 @@ client must have `root` access to interact with the daemon. If a group named
 `docker` exists on your system, `docker` applies ownership of the socket to the
 group.
 
+To interact with the remote API using [cURL](https://curl.haxx.se/download.html).
+You run `curl` against the socket where the Docker daemon is listening. For this
+reason, must use cURL 7.40 (or greater) as these versions have the
```

Comment:
To connect to the Docker daemon with `cURL` you need to use version 7.40 or later, as these versions provide the `--unix-socket` flag.